client/utils/index.js

```diff
@@ -259,3 +259,9 @@ export const isEventAllDay = (startingDate, endingDate) => {
     return startingDate.isSame(startingDate.clone().startOf('day')) &&
         endingDate.isSame(endingDate.clone().endOf('day').seconds(0).milliseconds(0))
 }
+
+export const isLockedByMe = (item, sessionDetails) => (
```

Comment:
I would have created a selector for this instead. Or a selector factory with the item as parameter